### PR TITLE
Molecule test framework added.

### PR DESCRIPTION
**Pull request by bbaassssiiee**
I like to use molecule as a test framework form my roles as a pre-flight check, It bundles all kinds of tests that help make Ansible Galaxy improve. I caught some issues during the implementation, see the separate commits, mostly minor.

[https://molecule.readthedocs.io/](https://molecule.readthedocs.io/)

### CODE REVIEW

File: molecule/default/molecule.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    image: centos:7
+    privileged: true
+  - name: gcloud-fedora
+    image: fedora:latest
+    privileged: true
+  - name: gcloud-ubuntu
+    image: ubuntu:latest
+    privileged: true
+  - name: gcloud-debian
+    image: debian:buster
```
Comment (claudeleveille) [inline molecule/default/molecule.yml]: I might bump this to stretch later.

File: defaults/main.yml [hunk 2]
```diff
@@ -1,3 +1,4 @@
 ---
-gcloud_disable_usage_reporting: yes
-gcloud_disable_update_check: no
+gcloud_disable_usage_reporting: true
+gcloud_disable_update_check: false
+gcloud_site: packages.cloud.google.com/apt
```
Comment (claudeleveille) [inline defaults/main.yml]: I'm not sure about this variable, doesn't seems like there'd ever be a reason to expose this in a default.

Comment (bbaassssiiee) [reply defaults/main.yml]: Corporate use, internal repositories.

**Comment by claudeleveille** [comment]
Overall, this is a very nice PR, but I was previously using molecule before I got rid of it. It doesn't seem to want to work on either of my MacBooks and it gives weird error messages that come from who-knows-which one of its far too numerous layers of indirection... Or so I thought when I was misusing it. Your PR has changed my mind.

This patch-set has inspired me go forth and implement this on my other roles and put this all into CI.

Thank you!

**Comment by bbaassssiiee** [comment]
Please tag and release...

**Comment by claudeleveille** [comment]
That's coming, I'm setting up CI.

**Comment by claudeleveille** [comment]
> That's coming, I'm setting up CI.

Since I know it works, I've released v0.1.4 a bit early. Now time to go automate this stuff.

**Pull request merged**